PF/Assignment2/pyramid2Test.java

```diff
@@ -0,0 +1,52 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+
+public class pyramid2Test {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void test1() {
+		assignment2 as1=new assignment2();
+		String output=as1.spaces(4,5);
+		assertEquals("","    ",output);
```

Comment:
ACTION: Proper message should be displayed it will be easy to understand in case of assertion failure.
you should compare length of the string as it will be easy to understand when the string contains all spaces.